lib/src/bloc/dice_bloc.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:async';
+import 'dart:math';
+import 'package:dice_roller/utils/constants.dart';
+import 'i_dice_bloc.dart';
+
+class DiceBloc extends IDiceBloc {
+  List<int> _values = [];
+
+  StreamController<List<int>> _numbersStreamController =
+      StreamController<List<int>>.broadcast();
+
+  StreamController<String> _scoreStreamController =
+      StreamController<String>.broadcast();
+
+  @override
+  void initialize() {}
+
+  @override
+  void dispose() {
+    _numbersStreamController.close();
+    _scoreStreamController.close();
+  }
+
+  @override
+  Stream<List<int>> get numbersStream => _numbersStreamController.stream;
+
+  @override
+  Stream<String> get scoreStream => _scoreStreamController.stream;
+
+  @override
+  void diceRoll() {
+    _values.clear();
+    int sumOfTheScore = 0;
+    int diceValue = 0;
+    int i = 0;
```

Comment:
Remember that although the method is public, the variables you use internally are private, therefore they should start with '_'

Reply:
Fixed